### DISCUSSION TITLE
Price Adjustment for Enchanted Golden Apples

### DISCUSSION THREAD

**Discussion by Cedric-Vii** [Stardust-Labs-MC/StardustMC#6] [Ideas]
Currently, Enchanted Golden Apples sell to the Adminshop for the same price as Hearts of the Sea, despite being considerably harder/more complicated to acquire. Suggesting a slight increase to Enchanted Golden Apples' sell value to better match their higher barrier for acquisition.

**Comment by ReiTheGoat01** [comment]
Thats fair maybe an increase to 50-60

**Reply by aisukuma** [reply]
Seems like a fair price

**Comment by EshXKun** [comment]
80

**Comment by aisukuma** [comment]
Enchanted Golden Apples can now be sold for 70 Starz
